mobile-app/src/screens/RemoteScreen.tsx

```diff
@@ -12,23 +12,45 @@ export function RemoteScreen({ navigation, route }: RemoteScreenProps) {
 
   React.useEffect(() => {
     getGame(route.params.gameId).then(setGameState);
-    const interval = setInterval(() => getGame(route.params.gameId).then(setGameState), REFRESH_TIMER);
-    return () => { clearInterval(interval) };
+    const interval = setInterval(
+      () => getGame(route.params.gameId).then(setGameState),
+      REFRESH_TIMER,
+    );
+    return () => {
+      clearInterval(interval);
+    };
   }, []);
 
   const onCellPress = (coordinates: Coordinates) => {
-    return gameState.currentPlayerTurnToPlay ? updateGame(gameState.game, coordinates).then(setGameState) : () => { }
-  }
+    return gameState.currentPlayerTurnToPlay
+      ? updateGame(gameState.game, coordinates).then(setGameState)
+      : () => {};
+  };
 
-  return gameState ?
-    gameState.readyToPlay ? <View style={{ flex: 1, flexDirection: 'column', alignItems: 'center', justifyContent: 'center' }}>
-      <ScrollView horizontal>
-        <Board gameState={gameState.game.state} onCellPress={onCellPress} />
-      </ScrollView>
-    </View>
-      : <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+  return gameState ? (
+    gameState.readyToPlay ? (
+      <View
+        style={{
+          flex: 1,
+          flexDirection: 'column',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}
+      >
+        <ScrollView horizontal>
+          <Board gameState={gameState.game.state} onCellPress={onCellPress} />
+        </ScrollView>
+      </View>
+    ) : (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
         <Text style={{ fontSize: 24 }}> Waiting for an opponent..</Text>
-        <Text style={{ fontSize: 18 }}> Use this join code : {route.params.gameId}</Text>
-      </View >
-    : <View />
+        <Text style={{ fontSize: 18 }}>
+          {' '}
+          Use this join code : {route.params.gameId}
```

Comment:
I'm a bit surprised by this code, I'll check real quick if it still runs on my machine...

Reply:
Tested working just fine 🙂

Reply:
Yes it's work !